pom.xml

```diff
@@ -739,6 +805,43 @@
             <scope>test</scope>
         </dependency>
 
+        <!-- Upper bound fix:
+                These dependencies are defined in core but other plugins brings in higher version of dependencies
+                resulting in upper bound check failure.
+
+                After core is updated to depend on later version of these dependencies AND blueocean depends on that core,
+                 we are going to list these dependencies here.
+        -->
+        <dependency>
+            <groupId>org.jenkins-ci.plugins.icon-shim</groupId>
+            <artifactId>icon-set</artifactId>
```

Comment:
Ask @tfennelly for advice on https://github.com/jenkinsci/plugin-pom/pull/72.

Reply:
This error is because we have **test** dependency on matrix-auth, that brings in different version of icon-set. Looks like this is a bug in matrix-auth, icon-set dependency comes from core and should not be overridden in matrix-auth. I am going to exclude icon-set from matrix-auth dependency for now.

```
Require upper bound dependencies error for org.jenkins-ci.plugins.icon-shim:icon-set:1.0.5 paths to dependency are:
+-io.jenkins.blueocean:blueocean-rest-impl:1.3.0-beta-3-SNAPSHOT
  +-org.jenkins-ci.main:jenkins-core:2.46.3
    +-org.jenkins-ci.plugins.icon-shim:icon-set:1.0.5
and
+-io.jenkins.blueocean:blueocean-rest-impl:1.3.0-beta-3-SNAPSHOT
  +-org.jenkins-ci.plugins:matrix-auth:1.7
    +-org.jenkins-ci.plugins.icon-shim:icon-shim:2.0.3
      +-org.jenkins-ci.plugins.icon-shim:icon-set:2.0.3
]
```

Reply:
@daniel-beck may know.